src/shared/Textarea/Textarea.jsx

```diff
@@ -0,0 +1,39 @@
+import { texareaProps } from './Texarea.props';
+import { TextareaStyled } from './Textarea.styled';
+
+/**
+ * Textarea component for rendering a multiline input field.
+ *
+ * @component
+ * @param {string} value - The current value of the textarea saved in state.
+ * @param {function} onChange - The callback function to handle changes to the current value.
+ * @param {string} placeholder - The placeholder text for the textarea.
+ * @param {number} minRows - The minimum number of rows for the textarea.
+ * @param {number} maxRows - The maximum number of rows for the textarea.
+ * @param {number} maxLength - The maximum length of the text in the textarea.
+ * @returns {JSX.Element}
+ */
+
+export const Textarea = ({
```

Comment:
Could you please name this component like I did before so that it has the same template names (AppButton, AppTextInput, etc).
In your case, that would be AppTextArea.jsx

And we have a centralized file to get such reusable components is shared/index.js. So you can add the line 
export { default as AppTextarea} from './TextArea/AppTextArea.jsx';